.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
```

Comment:
Is this an idea for the project ? or is this about the new feature?

Reply:
The default statement on GitHub is like that :). We can change the wording later on all repositories.